### PR TITLE
Rename Learning WG to Rustc Dev Guide WG

### PR DESCRIPTION
**Pull request by spastorino**
Part of #265.

### CODE REVIEW

File: posts/inside-rust/2019-10-28-rustc-learning-working-group-introduction.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
 layout: post
-title: "The Rustc Learning Working Group - An Introduction"
+title: "The Rustc Rustc Dev Guide Working Group - An Introduction"
```
Comment (chrissimpkins) [inline posts/inside-rust/2019-10-28-rustc-learning-working-group-introduction.md:3]: Duplicated Rustc
```suggestion
title: "The Rustc Dev Guide Working Group - An Introduction"
```

File: posts/inside-rust/2019-10-28-rustc-learning-working-group-introduction.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 author: Amanjeev Sethi
-description: "introduction learning working group useful links"
-team: the rustc learning working group <https://www.rust-lang.org/governance/teams/compiler#wg-learning>
+description: "introduction rustc dev guide working group useful links"
+team: the rustc rustc dev guide working group <https://www.rust-lang.org/governance/teams/compiler#wg-rustc-dev-guide>
```
Comment (chrissimpkins) [inline posts/inside-rust/2019-10-28-rustc-learning-working-group-introduction.md:6]: Duplicated rustc
```suggestion
team: the rustc dev guide working group <https://www.rust-lang.org/governance/teams/compiler#wg-rustc-dev-guide>
```

File: posts/inside-rust/2020-03-13-rename-rustc-guide.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: post
 title: "The rustc-guide is now rustc-dev-guide"
 author: mark-i-m
 description: "the guide has been renamed"
-team: the rustc learning working group <https://www.rust-lang.org/governance/teams/compiler#wg-learning>
+team: the rustc rustc dev guide working group <https://www.rust-lang.org/governance/teams/compiler#wg-rustc-dev-guide>
```
Comment (chrissimpkins) [inline posts/inside-rust/2020-03-13-rename-rustc-guide.md:6]: duplicated rustc
```suggestion
team: the rustc dev guide working group <https://www.rust-lang.org/governance/teams/compiler#wg-rustc-dev-guide>
```

File: posts/inside-rust/2019-10-15-compiler-team-meeting.md [hunk 4]
```diff
@@ -26,11 +26,11 @@ Each week, we have general announcements from the team followed by check-ins fro
 
 ## Working group sync
 
-### [wg-learning](https://rust-lang.github.io/compiler-team/working-groups/learning/)
+### [wg-rustc-dev-guide](https://rust-lang.github.io/compiler-team/working-groups/learning/)
```
Comment (spastorino) [inline posts/inside-rust/2019-10-15-compiler-team-meeting.md:29]:
```suggestion
### [wg-rustc-dev-guide](https://rust-lang.github.io/compiler-team/working-groups/rustc-dev-guide/)
```

**Comment by spastorino** [comment]
The rename is done already and this is ready to be merged.

**Pull request merged**